src/Core/src/Handlers/HybridWebView/HybridWebView.js

```diff
@@ -18,13 +18,13 @@
             window.dispatchEvent(event);
         }
         // Determine the mechanism to receive messages from the host application.
-        if (window.chrome?.webview?.addEventListener) {
+        if (window.chrome && window.chrome.webview && window.chrome.webview.addEventListener) {
```

Comment:
The Android Chromeis weird. It suports async functions ahead of its time, but then also does not support `?.`. I am at a loss.